src/components/VSlider/VSlider.js

```diff
@@ -31,6 +31,7 @@ export default {
   props: {
     alwaysDirty: Boolean,
     inverseLabel: Boolean,
+    hideLabel: Boolean,
```

Comment:
This seems unnecessary, if you just add `...this.$attrs` to the bottom of the attrs object then the user can apply `aria-label` themselves.

Reply:
- `label="something"` -> `<label>something</label>` + `aria-label="something"`
- `aria-label="something"` -> `aria-label="something"` + no `<label>`
- `label="something" aria-label="something else` -> `<label>something</label>` + `aria-label="something else"`

Reply:
will do